bin/bcftools_merge__delly.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+
+# Create temporary directory
+tmp_dir="interim"
+
+mkdir $tmp_dir
+
+# Extract unique sample prefixes from filenames
+prefixes=( $(for file in *.bcf.gz *.vcf.gz; do basename "$file" | cut -d '.' -f 1,2; done | sort -u) )
```

Comment:
I've extracted this script, contributed by @vrennie into a separate testable script in the `bin` folder.